oshc/main/static/main/css/app.css

```diff
@@ -435,29 +435,18 @@ body {
 .timeline ul li {
   list-style-type: none;
   position: relative;
-  width: 6px;
+  width: 0.5%;
```

Comment:
..and the magic of using `%` or `em` instead of specifying `px`. Nice you picked `%` over `em`. 👍 I checked off todo from [here](https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues/207) too. 🎉 

You might want to try it on bigger screens too, the odd child's triangle will move away from the rectangle. Let me know if you have questions for this or I'll take care of this before merging.